libs/go/msdagent/fsutil/fsutil.go

```diff
@@ -0,0 +1,47 @@
+package fsutil
```

Comment:
is it possible to extend https://github.com/AthenZ/athenz/blob/master/libs/go/sia/file/file.go and use that instead of declaring a diff fsutil.go?